### PR TITLE
Bug/153 cannot logout during sso

### PR DESCRIPTION
**Pull request by theblockstalk**

### CODE REVIEW

**Review by sadiabbasi — APPROVED** [review]

File: src/accounts/pages/Login.tsx [hunk 1]
```diff
@@ -57,11 +57,8 @@ export default function Login() {
   const [app, setApp] = useState<App>();
   const navigation = useNavigate();
   const communication = useUserStore((state) => state.communication);
-  const setUser = useUserStore((state) => state.setUser);
-  const isLoggedIn = useUserStore((state) => state.isLoggedIn);
-  const logout = useUserStore((state) => state.logout);
   const errorStore = useErrorStore();
-  let user: ExternalUser | undefined;
+  const { user, setUser, isLoggedIn, logout } = useUserStore();
```
Comment (sadiabbasi) [inline src/accounts/pages/Login.tsx:61]: where this logout is used?

File: src/accounts/pages/Login.tsx [hunk 2]
```diff
@@ -204,18 +201,15 @@ export default function Login() {
...
       const externalLoginRequest = await UserApps.onRedirectLogin();
 
       getAppDetails(externalLoginRequest);
       const requests = [externalLoginRequest];
       let loginToCommunication: AuthenticationMessage;
 
-      if (loggedIn === false) {
+      if (!user) {
         const { loginRequest, loginToCommunication: loginToCommunicationVal } =
           (await api.ExternalUser.loginWithTonomy({
```
Comment (sadiabbasi) [inline src/accounts/pages/Login.tsx:214]: before redirect don't we need to logout user?

File: src/accounts/pages/Login.tsx (hunk 1 above)
Comment (theblockstalk) [reply src/accounts/pages/Login.tsx:61]: L322

File: src/accounts/pages/Login.tsx (hunk 2 above)
Comment (theblockstalk) [reply src/accounts/pages/Login.tsx:214]: no I don't think so. only if user clicks to logout (`onLogout()`)

**Pull request merged**